DBTools/python/osusub_cfg.py

```diff
@@ -186,13 +207,18 @@ def getSiblingList(sibList, runList, siblingDataset):
   fin = open(sibList, 'r')
   data = json.load(fin)
 
+  aux_int = 0
```

Comment:
I added this "counter" to keep track of number of files when the number of events is not -1, but I am not sure if it was the best way to go. Anyone has any comments/suggestions on how to change it, or to leave it as is?

Reply:
I think this is fine, my only other suggestions would be `for filename in runlist[:len(data)]` basically the same thing but no counter

Reply:
@carriganm95 I am not really sure this would work because sometimes runList includes files that don't contain siblings and should be skipped (I am seeing this in the `allLocal` case), and then the number of siblings doesn't take those skips into account. I am doing some more tests to check it, and maybe will commit another change to this part of the config